server/src/main/java/com/linecorp/centraldogma/server/internal/mirror/GitMirror.java

```diff
@@ -395,6 +398,15 @@ private GitWithAuth openGit(File workDir) throws Exception {
         }
     }
 
+    private static String generateCommitDetail(String abbrId, RevCommit headCommit) {
+        final String newLine = System.lineSeparator();
```

Comment:
I think we need to use `\n` instead of the line separator so that the detail does not have a different message depending on the OS.

Reply:
Thanks @minwoox, as I understand from https://docs.oracle.com/javase/9/docs/api/java/lang/System.html#lineSeparator--
We should use `System.lineSeparator()`, it will auto-generate line separator on Unix or Window. 
I am just concerned that if we use `\n`, how will it run on Window because line separator on Window is `\r\n`?
Feel free to correct me if I am wrong 🚀

Reply:
If the file has different separators depending on the OS, that would cause certain problems. So we usually use just `\n` and it's automatically handleed via `core.autocrlf`. 😄 
This [link](https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration#_formatting_and_whitespace) explains well about the problem. 😄

Reply:
Ah, ok, I understand opposite. 
Many thanks😊